docs/INSTALLATION.MD

```diff
@@ -1,3 +1,11 @@
+# Installation
+
+## Pre-requisites
+
+# git-secrets
+git-secrets scans commits, commit messages, and --no-ff merges to prevent adding secrets into your git repositories.
+For installing, follow steps at https://github.com/awslabs/git-secrets#installing-git-secrets.
+
```

Comment:
some issue with the size of the headings here. the `git-secrets` is larger than `pre-requisites` and so on